openmp/runtime/src/CMakeLists.txt

```diff
@@ -292,6 +294,7 @@ if(WIN32)
   set(LIBOMP_IMP_LIB_TARGET omp)
   set(LIBOMP_GENERATED_DEF_FILE ${LIBOMP_LIB_NAME}.def)
   add_custom_target(libomp-needed-def-file DEPENDS ${LIBOMP_GENERATED_DEF_FILE})
+  set_target_properties(libomp-needed-def-file PROPERTIES FOLDER "OpenMP/Codegenning")
```

Comment:
`Codegenning` is really weird. I don't see this word anywhere else in LLVM project. Probably just `OpenMP/CodeGen`.

Reply:
For LLVM/Clang/MLIR the folder is already named ["Tablegenning"](https://github.com/llvm/llvm-project/blob/31f1590e4fb324c43dc36199587c453e27b6f6fa/llvm/cmake/modules/TableGen.cmake#L170) since forever. Since tablegen is not involved here, I found "Codegenning" more appropriate.